src/python/CMakeLists.txt

```diff
@@ -45,6 +45,13 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
+# FIXME: Windows client currently does not support GPU tensors.
+# For simplicity, we will override this option here.
+if(WIN32 AND TRITON_ENABLE_GPU)
+  message("GPU shared memory is not currently supported by the Windows python client. Forcing TRITON_ENABLE_GPU to false.")
+  set(TRITON_ENABLE_GPU OFF CACHE BOOL "GPU disabled" FORCE)
```

Comment:
Perhaps the impact of placing this here is too extreme. Maybe it would be better being placed in the cmakelist for utils?

Reply:
I think the best practice would be too throw an error when trying to configure `TRITON_ENABLE_GPU` to 1 in `src/python/library/tritonclient/utils/CMakeLists.txt`

Another good practice is to keep the checks as close as possible to the location of impact. We don't support GPU cuda shared memory on windows which comes from the utilities. However, there might be some features that can be enabled by "TRITON_ENABLE_GPU" in the library on windows.